app/assets/stylesheets/pages/solution-fixing.scss

```diff
@@ -1,18 +0,0 @@
-@import 'mixins';
-
-#my-solution-unlocked-page,
-#my-solution-page,
-#mentor-solution-page {
-    .panels {
-        .rhs {
-            overflow:hidden;
-            margin-top:30px;
-            margin-bottom:30px;
-            @include width-medium() {
```

Comment:
This bit handles the mobile-responsiveness of panels. Does this PR break that, or is it fixed somewhere else that I don't missed?

Reply:
Thanks. Good catch.